core/src/main/resources/AppConstants.properties

```diff
@@ -29,9 +29,8 @@ instance.timestamp=
 #Deprecated should be automatically determined by the instance.version and instance.timestamp properties (typically automatically generated and stored in BuildInfo.properties)
 instance.build_id=not-used
 
-configurations.names.application=IAF_WebControl,${configurations.names},IAF_Util
+configurations.names.application=IAF_WebControl,IAF_Util,${configurations.names}
```

Comment:
Waarom moet dit, en hoefde het eerst niet?

Reply:
Vroeger moest eerst de 'main/hoofd' configuratie (die op het classpath staat) ingeladen worden met een jmsRealm er in. Nu hoeft dat niet meer. Ik vind dat het mooier is als de IAF configuraties eerst inladen.

Reply:
Ah, zie nu wat er gebeurt is. Is alleen volgorde verandering. Ik had verkeerd gekeken, en dacht dat ${configurations.names} _erbij_ was gekomen, maar dus alleen maar verplaatst.